.env.e2e

```diff
@@ -1,5 +1,8 @@
-VUE_APP_BACKEND_URL=https://test-tipping.aeternity.art
+VUE_APP_BACKEND_URL=https://testnet.superhero.aeternity.art
 VUE_APP_NODE_URL=https://testnet.aeternity.io
 VUE_APP_EXPLORER_URL=https://testnet.aeternal.io
 VUE_APP_CONTRACT_V1_ADDRESS=ct_2GRP3xp7KWrKtZSnYfdcLnreRWrntWf5aTsxtLqpBHp71EFc3i
 VUE_APP_CONTRACT_V2_ADDRESS=
+VUE_APP_CONTRACT_V3_ADDRESS=ct_WscpdLQf6ZZxoVqrsEwUwmuAEdzEkJii5W5TzG84rVgHeK6BW
+VUE_APP_IMGUR_API_CLIENT_ID=eafea0a779e4039
+VUE_APP_GIPHY_API_KEY=P16yBDlSeEfcrJfp1rwnamtEZmQHxHNM
```

Comment:
more specific configs inherit the general one, so no need to specify `VUE_APP_IMGUR_API_CLIENT_ID`, `VUE_APP_GIPHY_API_KEY ` as far as they are the same in `.env`